pyproject.toml

```diff
@@ -114,7 +114,8 @@ Issues = "https://github.com/scikit-build/scikit-build-core/issues"
 "distutils.commands".build_cmake = "scikit_build_core.setuptools.build_cmake:BuildCMake"
 "distutils.setup_keywords".cmake_source_dir = "scikit_build_core.setuptools.build_cmake:cmake_source_dir"
 "distutils.setup_keywords".cmake_args = "scikit_build_core.setuptools.build_cmake:cmake_args"
-"setuptools.finalize_distribution_options".scikit_build_entry = "scikit_build_core.setuptools.build_cmake:finalize_distribution_options"
+"distutils.setup_keywords".cmake_extension = "scikit_build_core.setuptools.build_cmake:cmake_extension"
+"setuptools.finalize_distribution_options".scikit_build_entry = "scikit_build_core.setuptools.build_cmake:cmake_command"
```

Comment:
```suggestion
"setuptools.finalize_distribution_options".scikit_build_entry = "scikit_build_core.setuptools.build_cmake:finalize_distribution_options"
```

Reply:
This is not compulsory as far as I can see. But I'll follow any convention you want.

Reply:
IMO it reduces having to look this up, as the only way this is called is via entry point.